admin/services/check-window-v2.service.js

```diff
@@ -89,43 +89,45 @@ checkWindowV2Service.prepareSubmissionData = (requestData, checkWindowId = null)
   if (checkWindowId) {
     checkWindowData.id = checkWindowId
   }
+  const utcOffsetInHours = moment().utcOffset() / 60
```

Comment:
what unit is `moment().utcOffset()` ?

Reply:
@GuyHarwood minutes

Reply:
https://momentjs.com/docs/#/manipulating/utc-offset/